src/Talis/Persona/Client/Login.php

```diff
@@ -4,28 +4,68 @@
 
 class Login extends Base
 {
+    // Constants Related to simple-nonce library
+    const NONCE_EXPIRY_TIME_SECONDS = 300; // Nonce expires after 5 minutes at which point the persion in logged out
+    const NONCE_REFRESH_TIME_SECONDS = 180; // Nonce is refreshed if it is older than the refresh time of 3 minutes
+    const LOGIN_STATE_ACTION = 'loginState';
+    const NONCE_SALT = 'This is the salt used for generating the nonce';
```

Comment:
Env var maybe? Though I'm not sure what the consequences of changing it would be...